### PR TITLE
listing vertex names in edgeprob

### PR DESCRIPTION
**Pull request by bdesmarais**
I revised the edgeprob function to include columns for the vertex names of i and j. This was in response to a request from Christiane Bjerglund. I updated the documentation as well.

### CODE REVIEW

File: R/interpretation.R [hunk 1]
```diff
@@ -286,30 +286,27 @@ setMethod("interpret", signature = className("mtergm", "btergm"),
...
   if (class(object) == "ergm") {
     tergm <- FALSE
-  } else if (class(object) %in% c("btergm", "mtergm")) {
+  }
+  else if (class(object) %in% c("btergm", "mtergm")) {
     tergm <- TRUE
-  } else {
+  }
+  else {
     stop(paste("The tieprob function is only applicable to ergm, btergm, and",
```
Comment (leifeld) [inline R/interpretation.R]: Write `edgeprob` instead of `tieprob`?

File: R/interpretation.R [hunk 2]
```diff
@@ -344,25 +342,32 @@ edgeprob <- function(object, verbose = FALSE) {
...
+                  "predicted probabilities are not valid and must be manually replaced",
+                  "by 0 or 1, respectively."))
   }
-  
-  # compute predicted probabilities
   cbcoef <- cbind(cf)
   chgstat <- dyads[, 2:(ncol(dyads) - 3)]
   lp <- apply(chgstat, 1, function(x) t(x) %*% cbcoef)
-  result <- c(1 / (1 + exp(-lp)))
+  result <- c(1/(1 + exp(-lp)))
```
Comment (leifeld) [inline R/interpretation.R:356]: Can you preserve spaces around operators?

File: R/interpretation.R [hunk 3]
```diff
@@ -344,25 +342,32 @@ edgeprob <- function(object, verbose = FALSE) {
...
+  # add in vertex ids
+  i.name <- numeric(nrow(dyads))
+  j.name <- numeric(nrow(dyads))
+  for (t in 1:length(l$networks)) {
+    vnames.t <- colnames(l$networks[[i]][,])
+    dyads.t <- dyads[which(dyads$t==t),]
```
Comment (leifeld) [inline R/interpretation.R]: Put spaces around `==`. Same multiple times below and after the `,` in `[,]`.

**Review by leifeld — CHANGES_REQUESTED** [review]
Hi Bruce,

Thanks a lot for starting this pull request. I have added a few comments, mostly about formatting, in my review. There is one more thing to take into account: the `edgeprob` function is used internally in the `marginalplot` function for creating marginal effects plots. This function may need to be updated as well because the labels may otherwise be interpreted as covariates. Would you be able to do that? I think it shouldn't be a lot of work.

Thanks,

Philip

**Comment by bdesmarais** [comment]
Thanks for the quick review, Philip! I corrected the warning message and spaces. I reviewed and tested marginalplot. I think the only way an issue would arise is if the user input "i.name" or "j.name" as one of the variable names in the call to marginalplot.

**Comment by leifeld** [comment]
Thanks, Bruce. I have just reviewed the list of files you are adding. Can you remove `btergm.Rproj` and `.Rhistory`? I don't think we need them.

**Comment by bdesmarais** [comment]
Just removed those files.

**Comment by leifeld** [comment]
Thanks. I have merged it!

**Pull request merged**